kolibri/plugins/management/assets/src/vue/user-page/index.vue

```diff
@@ -213,6 +212,14 @@
     height: 35px
     outline: none
 
+  .roster
+    width: 100%
+
+    /*Prevent lists that are too long*/
+    max-height: 300px
+    overflow: hidden
+    overflow-y: scroll
+
```

Comment:
I think they were fixes from the missing roster class of Tuesday's PR. It's likely they were added in because she noticed it was missing.